### ISSUE TITLE
JAWS/NVDA: Canceled appointments are not announced as canceled [Lvl 2]

### ISSUE THREAD

**Issue by outerpress** [department-of-veterans-affairs/va.gov-team#111652]
## Description

The canceled appointments are marked with strikethrough text to inform the user that an appointment has been canceled. This will not be announced by a screen reader.

● Page: Upcoming and Past Appointments Dashboard
● Device: Windows + JAWS / NVDA
● Orientation: Both
● Expected severity level: 2
● Expected experience standard: User can perceive all meaningful elements
● How to recreate:
○ Turn on JAWS / NVDA
○ Arrow or tab to the appointment controls
● Recommendation: Ensure that the canceled appointments are marked with text as canceled (e.g., “… amputation care appointment canceled”) to inform the user that the appointment has been canceled.

---

## Development Checklist
Background: the `toggle` toggle is on (remove if not needed) 

- [ ] Task 1
- [ ] Task 2

### Design Assets
- [Figma file]() (put N/A if not needed) 

### Developer Reference

- [Swagger File]() (remove if not needed)
- [File path]() (remove if not needed)

## Definition of Done
- [ ] All tasks criteria are met
- [ ] [Technical Documentation]() is updated (must confirm if needed/or remove) 
- [ ] [Feature Reference (GitHub)](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/health-care/appointments/va-online-scheduling/feature-reference) is updated
- [ ] UX - [Feature Reference (Figma)](https://www.figma.com/design/eonNJsp57eqfPqx7ydsJY9/Feature-Reference-%7C-Appointments-FE?node-id=1150-105402&t=zRNPBqgPDVU0dDP8-1) is updated

**Comment by outerpress** [comment]
Hi @mmutia - there are aria labels on all appointments in the list - can you confirm that JAWS is not announcing the aria labels on canceled appointments? Or is it not reading them on all appointments?

**Comment by outerpress** [comment]
@mmutia I haven't been able to test this in Jaws/NVDA, but using Narrator on Windows and VoiceOver on Mac the label reads correctly when tabbing through details links, i.e., "Details for canceled...."

Let me know if that's different from what you are experiencing/expected. FYA @jamiefiore @coforma-jamie
